modules/network/src/main/java/org/apache/ignite/internal/network/configuration/SslConfigurationValidatorImpl.java

```diff
@@ -99,9 +99,14 @@ private static void validateCiphers(ValidationContext<AbstractSslView> ctx, Abst
             Set<String> ciphers = Arrays.stream(ssl.ciphers().split(","))
                     .map(String::strip)
                     .collect(Collectors.toSet());
-            if (!supported.containsAll(ciphers)) {
-                ciphers.removeAll(supported);
-                ctx.addIssue(new ValidationIssue(ctx.currentKey(), "There are unsupported cipher suites: " + ciphers));
+
+            // If removeAll returns true, it means that there were at least some supported ciphers.
+            boolean haveSupported = ciphers.removeAll(supported);
+            if (!ciphers.isEmpty()) {
+                if (!haveSupported) {
+                    ctx.addIssue(new ValidationIssue(ctx.currentKey(), "None of the configured cipher suites are supported: " + ciphers));
+                }
+                LOG.info("Some of the configured cipher suites are unsupported: {}", ciphers);
```

Comment:
> We can log here not all ciphers but only those that are unsupported. It will be more informative for users.

Since we're doing `removeAll`, only unsupported ciphers are logged.